test/transaction_tests.erl

```diff
@@ -288,8 +288,11 @@ lock_wait_timeout1({Conn1, Conn2}) ->
     {aborted, Reason} = mysql:transaction(Conn2, fun () ->
         ok = mysql:query(Conn2, "UPDATE foo SET v = 42 WHERE k = 1")
```

Comment:
So, if we don't try to match this against `ok`, the transaction would not fail? I.e. it would return `{atomic, _}`?

Isn't it better to replace the `ok =` here with this ↓ then?

    ?assertMatch({error, {1205, _, <<"Lock wai", _/binary>>}},
                 mysql:query(...))`?

Reply:
Yeah, I'll update to better reflect the new behavior.